pkg/network/multus/multus_test.go

```diff
@@ -68,6 +68,53 @@ var _ = Describe("Multus", func() {
 			}),
 		)
 	})
+	Context("LookUpDefaultNetwork", func() {
+		type lookUpDefaultNetworkParams struct {
+			inputNetworks []v1.Network
+			expectNetwork v1.Network
+		}
```

Comment:
Same questions as before... why do we need a struct and not two arguments as input to the test table.

Reply:
It boils down to personal style, but I think this way has the benefit of separating the input "params" and the internal ones.
Do you think this way has bad traits?

Reply:
I am having hard time to understand it. I mean, the argument is called `params` which is indeed the function inputs, but in a structured manner instead of individual arguments.
If the member names are defined as the function arguments, the meaning and clarity is not degraded IMO.

> separating the input "params" and the internal ones

Could you please clarify what are these "internal ones"?

I am not sure, but are we trying here to find arguments on why we should use a single struct as an argument instead of multiple variables? `f(a, b, c)` ==>> `f(s)`
This seems to me like a general question, not limited to tests.

I just do not see the added value at the moment, only drawbacks in trying to understand what this struct is and why it was needed. We are adding here ` lookUpDefaultNetworkParams{` for each call and the definition of the struct once.

Reply:
>Could you please clarify what are these "internal ones"?

By that I mean variables that are created inside the test, as oppose to ones that are given to it.

>I am not sure, but are we trying here to find arguments on why we should use a single struct as an argument instead of multiple variables? f(a, b, c) ==>> f(s)
This seems to me like a general question, not limited to tests.

I wouldn't go as far as generalizing it to a general rule, but generally speaking, in tests (and specifically DescribeTable tests) I prefer using a struct if there is more than 1 params to insert. 
If you don't have a significant drawback then like I initially said - it boils down to style...